src/lib.rs

```diff
@@ -4,6 +4,7 @@
 pub mod client;
 pub mod errors;
 pub mod manifest;
+pub mod oci_annotations;
```

Comment:
Since this is called the `oci_distribution` crate, we should probably just call this module `annotations`. It also matches the rest of the module names that aren't prefaced by it

Reply:
good point, done